components/authz-service/server/v2/policy.go

```diff
@@ -119,6 +116,10 @@ func NewPoliciesServer(
 	}
 	srv.setVersion(v)
 
+	if err := srv.updateEngineStore(ctx); err != nil {
```

Comment:
ℹ️ this needs to happen _after_ `setVersion`, so the proper store will be updated in the process (v2 vs v2.1). (v1 is done per-call in its corresponding handlers)